pkgs/tools/security/sops/default.nix

```diff
@@ -2,16 +2,18 @@
 
 buildGoModule rec {
   pname = "sops";
-  version = "3.7.2";
+  version = "3.7.3";
 
   src = fetchFromGitHub {
     rev = "v${version}";
     owner = "mozilla";
     repo = pname;
-    sha256 = "sha256-NMuYMvaBSxKHvpqFkMfnMDvcXxTstqzracuSTT1VB1A=";
+    sha256 = "sha256-wN1ksLwD4G+fUhvCe+jahh1PojPk6L6tnx1rsc7dz+M=";
   };
 
-  vendorSha256 = "sha256-00/7O9EcGojUExJPtYWndb16VqrNby/5GsVs8Ak/Isc=";
+  vendorSha256 = "sha256-8IaE+vhVZkc9QDR6+/3eOSsuf3SYF2upNcCifbqtx14=";
+
+  ldflags = [ "-s" "-w" ];
 
   doCheck = false;
```

Comment:
Why not enable tests?

Reply:
The tests fails

<details>

```
nix-build -A sops
this derivation will be built:
  /nix/store/ylpafrkdzdc1rslvx8zbpjjav3v55ssb-sops-3.7.3.drv
building '/nix/store/ylpafrkdzdc1rslvx8zbpjjav3v55ssb-sops-3.7.3.drv'...

...

=== RUN   TestMasterKeysFromRecipientsWithMultiple
--- PASS: TestMasterKeysFromRecipientsWithMultiple (0.00s)
=== RUN   TestAge
    keysource_test.go:97: 
                Error Trace:    keysource_test.go:97
                Error:          Received unexpected error:
                                failed to open file: open go.mozilla.org/sops/v3/age/keys.txt: no such file or directory
                Test:           TestAge
    keysource_test.go:98: 
                Error Trace:    keysource_test.go:98
                Error:          Not equal: 
                                expected: []byte{0x61, 0x62, 0x63, 0x64, 0x65, 0x66, 0x67, 0x68, 0x69, 0x6a, 0x6b, 0x6c, 0x6d, 0x6e, 0x6f, 0x70, 0x71, 0x72, 0x73, 0x74, 0x75, 0x76, 0x77, 0x78, 0x79, 0x7a, 0x31, 0x32, 0x33, 0x34, 0x35, 0x36}
                                actual  : []byte(nil)
                                
                                Diff:
                                --- Expected
                                +++ Actual
                                @@ -1,5 +1,2 @@
                                -([]uint8) (len=32) {
                                - 00000000  61 62 63 64 65 66 67 68  69 6a 6b 6c 6d 6e 6f 70  |abcdefghijklmnop|
                                - 00000010  71 72 73 74 75 76 77 78  79 7a 31 32 33 34 35 36  |qrstuvwxyz123456|
                                -}
                                +([]uint8) <nil>
                                 
                Test:           TestAge
    keysource_test.go:97: 
                Error Trace:    keysource_test.go:97
                Error:          Received unexpected error:
                                failed to open file: open go.mozilla.org/sops/v3/age/keys.txt: no such file or directory
                Test:           TestAge
    keysource_test.go:98: 
                Error Trace:    keysource_test.go:98
                Error:          Not equal: 
                                expected: []byte{0x61, 0x62, 0x63, 0x64, 0x65, 0x66, 0x67, 0x68, 0x69, 0x6a, 0x6b, 0x6c, 0x6d, 0x6e, 0x6f, 0x70, 0x71, 0x72, 0x73, 0x74, 0x75, 0x76, 0x77, 0x78, 0x79, 0x7a, 0x31, 0x32, 0x33, 0x34, 0x35, 0x36}
                                actual  : []byte(nil)
                                
                                Diff:
                                --- Expected
                                +++ Actual
                                @@ -1,5 +1,2 @@
                                -([]uint8) (len=32) {
                                - 00000000  61 62 63 64 65 66 67 68  69 6a 6b 6c 6d 6e 6f 70  |abcdefghijklmnop|
                                - 00000010  71 72 73 74 75 76 77 78  79 7a 31 32 33 34 35 36  |qrstuvwxyz123456|
                                -}
                                +([]uint8) <nil>
                                 
                Test:           TestAge
--- FAIL: TestAge (0.00s)
=== RUN   TestAgeDotEnv
    keysource_test.go:130: 
                Error Trace:    keysource_test.go:130
                Error:          Received unexpected error:
                                failed to open file: open go.mozilla.org/sops/v3/age/keys.txt: no such file or directory
                Test:           TestAgeDotEnv
    keysource_test.go:131: 
                Error Trace:    keysource_test.go:131
                Error:          Not equal: 
                                expected: []byte{0x49, 0x4d, 0x41, 0x47, 0x45, 0x5f, 0x50, 0x52, 0x45, 0x46, 0x49, 0x58, 0x3d, 0x72, 0x65, 0x70, 0x6f, 0x2f, 0x73, 0x65, 0x72, 0x76, 0x69, 0x63, 0x65, 0x2d, 0xa, 0x41, 0x50, 0x50, 0x4c, 0x49, 0x43, 0x41, 0x54, 0x49, 0x4f, 0x4e, 0x5f, 0x4b, 0x45, 0x59, 0x3d, 0x4b, 0x36, 0x70, 0x66, 0x41, 0x57, 0x75, 0x55, 0x56, 0x4e, 0x44, 0x39, 0x46, 0x7a, 0x35, 0x53, 0x43, 0x37, 0x6a, 0x6d, 0x41, 0x36, 0x70, 0x66, 0x41, 0x57, 0x75, 0x55, 0x56, 0x4e, 0x44, 0x39, 0x46, 0x7a, 0x35, 0x53, 0x43, 0x37, 0x6a, 0x6d, 0x41, 0xa, 0x4b, 0x45, 0x59, 0x5f, 0x49, 0x44, 0x3d, 0x30, 0x30, 0x33, 0x36, 0x38, 0x33, 0x64, 0x37, 0x32, 0x31, 0x66, 0x32, 0x61, 0x65, 0x36, 0x38, 0x33, 0x64, 0x37, 0x32, 0x31, 0x66, 0x32, 0x61, 0x31, 0xa, 0x44, 0x4f, 0x4d, 0x41, 0x49, 0x4e, 0x3d, 0x66, 0x69, 0x6c, 0x65, 0x73, 0x2e, 0x31, 0x32, 0x37, 0x2e, 0x30, 0x2e, 0x30, 0x2e, 0x31, 0x2e, 0x6e, 0x69, 0x70, 0x2e, 0x69, 0x6f}
                                actual  : []byte(nil)
                                
                                Diff:
                                --- Expected
                                +++ Actual
                                @@ -1,13 +1,2 @@
                                -([]uint8) (len=147) {
                                - 00000000  49 4d 41 47 45 5f 50 52  45 46 49 58 3d 72 65 70  |IMAGE_PREFIX=rep|
                                - 00000010  6f 2f 73 65 72 76 69 63  65 2d 0a 41 50 50 4c 49  |o/service-.APPLI|
                                - 00000020  43 41 54 49 4f 4e 5f 4b  45 59 3d 4b 36 70 66 41  |CATION_KEY=K6pfA|
                                - 00000030  57 75 55 56 4e 44 39 46  7a 35 53 43 37 6a 6d 41  |WuUVND9Fz5SC7jmA|
                                - 00000040  36 70 66 41 57 75 55 56  4e 44 39 46 7a 35 53 43  |6pfAWuUVND9Fz5SC|
                                - 00000050  37 6a 6d 41 0a 4b 45 59  5f 49 44 3d 30 30 33 36  |7jmA.KEY_ID=0036|
                                - 00000060  38 33 64 37 32 31 66 32  61 65 36 38 33 64 37 32  |83d721f2ae683d72|
                                - 00000070  31 66 32 61 31 0a 44 4f  4d 41 49 4e 3d 66 69 6c  |1f2a1.DOMAIN=fil|
                                - 00000080  65 73 2e 31 32 37 2e 30  2e 30 2e 31 2e 6e 69 70  |es.127.0.0.1.nip|
                                - 00000090  2e 69 6f                                          |.io|
                                -}
                                +([]uint8) <nil>
                                 
                Test:           TestAgeDotEnv
--- FAIL: TestAgeDotEnv (0.00s)
=== RUN   TestAgeEnv
    keysource_test.go:153: 
                Error Trace:    keysource_test.go:153
                Error:          Received unexpected error:
                                open go.mozilla.org/sops/v3/age/keys.txt: no such file or directory
                Test:           TestAgeEnv
    keysource_test.go:159: 
                Error Trace:    keysource_test.go:159
                Error:          Received unexpected error:
                                no secret keys found
                Test:           TestAgeEnv
    keysource_test.go:160: 
                Error Trace:    keysource_test.go:160
                Error:          Not equal: 
                                expected: []byte{0x61, 0x62, 0x63, 0x64, 0x65, 0x66, 0x67, 0x68, 0x69, 0x6a, 0x6b, 0x6c, 0x6d, 0x6e, 0x6f, 0x70, 0x71, 0x72, 0x73, 0x74, 0x75, 0x76, 0x77, 0x78, 0x79, 0x7a, 0x31, 0x32, 0x33, 0x34, 0x35, 0x36}
                                actual  : []byte(nil)
                                
                                Diff:
                                --- Expected
                                +++ Actual
                                @@ -1,5 +1,2 @@
                                -([]uint8) (len=32) {
                                - 00000000  61 62 63 64 65 66 67 68  69 6a 6b 6c 6d 6e 6f 70  |abcdefghijklmnop|
                                - 00000010  71 72 73 74 75 76 77 78  79 7a 31 32 33 34 35 36  |qrstuvwxyz123456|
                                -}
                                +([]uint8) <nil>
                                 
                Test:           TestAgeEnv
--- FAIL: TestAgeEnv (0.00s)
FAIL
FAIL    go.mozilla.org/sops/v3/age      0.182s
FAIL
error: builder for '/nix/store/ylpafrkdzdc1rslvx8zbpjjav3v55ssb-sops-3.7.3.drv' failed with exit code 1;
       last 10 log lines:
       >                             - 00000000  61 62 63 64 65 66 67 68  69 6a 6b 6c 6d 6e 6f 70  |abcdefghijklmnop|
       >                               - 00000010  71 72 73 74 75 76 77 78  79 7a 31 32 33 34 35 36  |qrstuvwxyz123456|
       >                               -}
       >                             +([]uint8) <nil>
       >
       >              Test:           TestAgeEnv
       > --- FAIL: TestAgeEnv (0.00s)
       > FAIL
       > FAIL    go.mozilla.org/sops/v3/age      0.182s
       > FAIL
       For full logs, run 'nix log /nix/store/ylpafrkdzdc1rslvx8zbpjjav3v55ssb-sops-3.7.3.drv'.
```
</details>